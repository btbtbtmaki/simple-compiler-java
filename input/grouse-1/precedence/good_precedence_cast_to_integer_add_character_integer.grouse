main {

  print '. :int + 60 nl;
  print 'u :int + 14 nl;
  print '- :int + 40 nl;
  print 'Z :int + 1 nl;
  print 'p :int + 89 nl;
  print 'U :int + 68 nl;
  print 'm :int + 74 nl;
  print 'z :int + 67 nl;
  print 'r :int + 5 nl;
  print 'B :int + 71 nl;
  print 'G :int + 86 nl;
  print '" :int + 19 nl;

  print 36 + '* :int nl;
  print 83 + 'M :int nl;
  print 13 + 't :int nl;
  print 82 + 'y :int nl;
  print 46 + '! :int nl;
  print 63 + '" :int nl;
  print 12 + '[ :int nl;
  print 57 + 'J :int nl;
  print 51 + '^ :int nl;
  print 11 + 'W :int nl;
  print 22 + '` :int nl;
  print 91 + 'c :int nl;

  print '$ :int + '4 :int nl;
  print '. :int + '] :int nl;
  print '. :int + 't :int nl;
  print ') :int + '" :int nl;
  print '} :int + 'm :int nl;
  print 't :int + 'l :int nl;
  print 'L :int + 'P :int nl;
  print '? :int + '  :int nl;
  print 'v :int + 'e :int nl;
  print ') :int + '# :int nl;
  print 'F :int + ': :int nl;
  print 'y :int + 'P :int nl;

}

