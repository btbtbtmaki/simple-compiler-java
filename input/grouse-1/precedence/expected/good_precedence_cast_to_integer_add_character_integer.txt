106
131
85
91
201
153
183
189
119
137
157
53
78
160
129
203
79
97
103
131
145
98
118
190
88
139
162
75
234
224
156
95
219
76
128
201
